include/maps.h

```diff
@@ -60,28 +58,50 @@ enum eMapMode
  */
 struct s_map
 {
-    signed char map_no; /*!< Not used in code. */
-    uint8_t zero_zone;  /*!< Non-zero if zone 0 triggers an event */
-    uint8_t map_mode;   /*!< Draw order for layers, plus parallax settings (see eMapMode) */
-    uint8_t can_save;   /*!< Non-zero if Save is allowed in this map */
-    uint8_t tileset;    /*!< Which tile-set to use */
-    uint8_t use_sstone; /*!< Non-zero if sunstone works on this map */
-    uint8_t can_warp;   /*!< Non-zero if Warp is allowed in this map */
-    uint8_t extra_byte; /*!< Currently unused */
-    uint32_t xsize;     /*!< Map width in tiles (pixels / TILE_W) */
-    uint32_t ysize;     /*!< Map height in tiles (pixels / TILE_H) */
-    int32_t pmult;      /*!< Parallax multiplier */
-    int32_t pdiv;       /*!< Parallax divider */
-    int32_t stx;        /*!< Default start x-coord */
-    int32_t sty;        /*!< Default start y-coord */
-    int32_t warpx;      /*!< x-coord where warp spell takes you to (see special_spells()) */
-    int32_t warpy;      /*!< y-coord where warp spell takes you to (see special_spells()) */
-    int revision;       /*!< Internal revision number for the map file */
-    int extra_sdword2;  /*!< Not used */
-    string song_file;   /*!< Base file name for map song */
-    string map_desc;    /*!< Map name (shown when map first appears) */
-    KMarkers markers;   /*!< Marker array */
-    KBounds bounds;     /*!< Bound array and bound size */
```

Comment:
Doesn't really matter but I found the 'point-left' comment format easier to read

Reply:
I plan to re-add those 'point-left' comments. I've got a lot of reformatting, and I've been using a bunch of regex to match everything and didn't want to accidentally leave the comments off.